kafka-ui-react-app/src/components/Connect/Details/Actions/__tests__/Actions.spec.tsx

```diff
@@ -35,8 +35,11 @@ const expectActionButtonsExists = () => {
 };
 const afterClickDropDownButton = async () => {
   const dropDownButton = screen.getAllByRole('button');
-  expect(dropDownButton.length).toEqual(1);
-  await userEvent.click(dropDownButton[0]);
+  await userEvent.click(dropDownButton[1]);
```

Comment:
let catch this with its name as well instead of `screen.getAllByRole('button');` like `screen.getByText('Restart Failed Tasks');`

Reply:
there is a no name button it's just a drop down button